source/agora/test/Base.d

```diff
@@ -95,6 +95,8 @@ public import std.range;
 // To print messages to the screen while debugging a test
 public import std.stdio;
 
+const size_t genesis_validators = GenesisBlock.header.enrollments.count();
```

Comment:
For constants we use `CamelCase` (see also `TotalNodes` below for an example).

Also please add `public`. 👼